### ISSUE TITLE
24. update README file

### ISSUE THREAD

**Issue by arcan9** [the-collab-lab/tcl-54-smart-shopping-list#54]
## Summary
Write a ReadMe file for the project and fix some glaring errors with rendering.

### ReadMe sections:

- [x] Title + description
- [x] Why We built this: purpose & who it's for
- [x] Technologies used
- [x] Live Demo (github pages link)
- [x] Features ("User can...", etc.)
- [x] Stretch Features (if applicable)
- [x] Preview (GIF or screenshots of UI)
- [x] Testing/QA + System Requirements

### Other
- [x] Bug fixes
- [x] Center align the home page content and add padding before the title

Update 03/08: Error fixed.

~~Currently, our project in production gives this error and does not load properly.~~

![tcl-error](https://user-images.githubusercontent.com/102492480/222936343-8923f639-a9c0-4eee-8a9b-eff6865acbae.jpg)

~~Not sure if it's only me. But other teams' projects don't have this problem when I checked. Looking it up, it is related to this React error: https://reactjs.org/docs/error-decoder.html/?invariant=299~~

**Comment by alex-andria** [comment]
I'm thinking that the root issue may be from the confirm window. And the logo error may be from the deleted unused files.

**Comment by arcan9** [comment]
> I'm thinking that the root issue may be from the confirm window. And the logo error may be from the deleted unused files.

@alex-andria It seems like the app is mounting before the DOM is fully loaded. But modifying the mounting in `index.jsx` seems to solve it (hopefully for good. lol). And  yep, you're right about the deleted files. I removed the reference for them from the vite config file and it did the trick!

Btw, I realized we can't deploy the app without admin rights. So I guess we'll just have to be okay with our original link to the app.
